### PR TITLE
Removing CORS hack and adding a proper middleware for handling CORS headers

### LINKED ISSUE #24: Return proper CORS headers in all REST API responses
(Cloned from content-service repository, where it doesn't apply anymore)

The following middleware might be useful:
https://www.gorillatoolkit.org/pkg/handlers#CORS

## Aceptance criteria
Have proper CORS headers using some library

### PR DESCRIPTION
**Pull request by joselsegura**
# Description

Remove CORS headers hack and use a proper middleware for handling the CORS headers.

Fixes #24 

## Type of change
- Bug fix (non-breaking change which fixes an issue)
- Refactor (refactoring code, removing useless files)

## Testing steps
Regular CI

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -19,12 +19,17 @@ github.com/DataDog/datadog-go v2.2.0+incompatible/go.mod h1:LButxg5PwREeZtORoXG3
...
 github.com/OneOfOne/xxhash v1.2.2/go.mod h1:HSdplMjZKSmBqAxg5vPj2TmRDmfkzw+cTzAElWljhcU=
 github.com/OneOfOne/xxhash v1.2.7/go.mod h1:eZbhyaAYD41SGSSsnmcpxVoRiQ/MPUTjUdIIOT9Um7Q=
+github.com/PuerkitoBio/goquery v1.5.1/go.mod h1:GsLWisAFVj4WgDibEWF4pvYnkVQBpKBKeU+7zCJoLcc=
 github.com/RedHatInsights/cloudwatch v0.0.0-20200512151223-b0b55757a24b/go.mod h1:8l+HqU8iWM6hA9kSAHgY3ItSlpEsPr8fb2R0GBp9S0U=
 github.com/RedHatInsights/insights-content-service v0.0.0-20200610101541-f0c10d6d451d h1:0aXuHiiucADzC++1lOJs3u4k1UanSNX+wIcpnyi8Bi0=
 github.com/RedHatInsights/insights-content-service v0.0.0-20200610101541-f0c10d6d451d/go.mod h1:j3Mc+Xh2ospxJ+K2RIZjYlI6E6E4H2Qh3cgUzhstrfQ=
 github.com/RedHatInsights/insights-operator-utils v1.0.1/go.mod h1:gRzYBMY4csuOXgrxUuC10WUkz6STOm3mqVsQCb+AGOQ=
 github.com/RedHatInsights/insights-operator-utils v1.0.2-0.20200610143236-c868b2f93d2a h1:/aAv1hq0ZFLDIxfmdTR23cm/B6nHEOVlWxOreOMqr80=
 github.com/RedHatInsights/insights-operator-utils v1.0.2-0.20200610143236-c868b2f93d2a/go.mod h1:0rhk13kn0BB+yKdfZjpMoQy0lXdXY3i4NJ1xGwGMcII=
+github.com/RedHatInsights/insights-operator-utils v1.1.0 h1:Bhoqqopchg4OgbDTx72PVQUuFlymyMyQPYEqN0YELP0=
```
Comment (tisnik) [inline go.sum:29]: This looks like a leftover.

File: go.sum [hunk 2]
```diff
@@ -19,12 +19,17 @@ github.com/DataDog/datadog-go v2.2.0+incompatible/go.mod h1:LButxg5PwREeZtORoXG3
...
+github.com/RedHatInsights/insights-operator-utils v1.1.0/go.mod h1:PHoCR0njiPXVArw+ooowqZgBjozu1h+8mYDoRoPVh6g=
```
Comment (tisnik) [inline go.sum:30]: dtto

Comment (joselsegura) [reply go.sum:29]: I merged master and rm go.sum, but this line remains here ???

Comment (tisnik) [reply go.sum:29]: probably wrong transitional dependencies?

Comment (joselsegura) [reply go.sum:29]: Yes, I guess content-service is depending on 1.1.0, but IDK who is depending on 1.0.2

Comment (joselsegura) [reply go.sum:29]: Aggregator is the one! :-D

Comment (tisnik) [reply go.sum:29]: nice catch!

**Pull request merged**